MAUI/DataGrid/pull-to-refresh.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: PullToRefresh in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about PullToRefresh support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control and more.
+platform: MAUI
+control: SfDataGrid
+documentation: UG
+---
+
+# PullToRefresh in MAUI DataGrid (SfDataGrid)
```

Comment:
Pull To Refresh in MAUI DataGrid (SfDataGrid)

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/d6a1e9a7-5190-47d6-ac4d-7f83b73cff6a)
Had updated as discussed.